server/tournaments/index.ts

```diff
@@ -749,7 +749,13 @@ export class Tournament extends Rooms.RoomGame {
 		if (matchTo) {
 			matchTo.isBusy = false;
 			const matchRoom = matchTo.inProgressMatch!.room;
-			matchRoom.parent = null;
+			if (matchRoom.parent) {
```

Comment:
Won't this still crash with `property 'parent' does not exist on undefined` or something like that?

Reply:
i figured it'll be fine since the current implementation is setting ``matchRoom.parent = null`` without checking if ``matchRoom`` even existed i could add the ``?`` to ``matchRoom`` though to be safe though

Reply:
let me add a ``?`` to be safe

Reply:
actually should be fine considering how much matchRoom is called without making sure it exists but i can add ``?`` if someone wants me to